rosbag2_performance/rosbag2_benchmarking/rosbag2_benchmarking/dummy_publishers.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020, Robotec.ai sp. z o.o.
```

Comment:
why do I need this? What's the difference to `ros2 topic pub <topic> <msg_type>`?

Reply:
Launching `ros2 topic pub` looked pretty slow, what is more, launching 1000 instances was way slower :smiley: